src/components/Measures/LockdownMeasures.js

```diff
@@ -34,6 +39,11 @@ const measures = [
     description:
       'Restaurants sowie Bars, Clubs, Diskotheken, Kneipen und ähnliche Einrichtungen bleiben geschlossen. Davon ausgenommen sind die Lieferung und Abholung von Speisen für den Verzehr zu Hause.',
   },
+  {
+    title: 'Homeoffice',
+    description:
+      'Arbeitgeber*Innen müssen künftig überall, wo es möglich ist, Arbeiten im Homeoffice ermöglichen.',
```

Comment:
Shouldn't it be "Arbeitgeber*innen"? I am not sure but I think I heard that : is better for screen readers, so "Arbeitgeber:innen". Totally okay if we keep it like this though!